### PR TITLE
API-1371 events api overview

### PR DESCRIPTION
**Pull request by meganelepalud-akeneo**

### CODE REVIEW

File: content/events-api/overview.md [hunk 1]
```diff
@@ -2,6 +2,22 @@
...
+The feature is working on simple products and product models and variants. 
 
-// todo
\ No newline at end of file
+::: info 
+For more information, please read our help center article [What is the event subscription?](https://help.akeneo.com/pim/serenity/articles/what-is-the-event-subscription.html) to discover which type of events the Akeneo PIM triggers and in what cases. 
+It explains to the Akeneo PIM users how to setup their PIM to use an extension that uses events to synchronize the Akeneo PIM product data with other applications. 
+::: 
+
+## The event loop
```
Comment (fitn) [inline content/events-api/overview.md:13]:
```suggestion
## The event API loop
```

We should take care of the words we are using everywhere (event, event API, event subscription, etc.).

File: content/events-api/receiving-events.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Request URL receives an HTTP POST containing data in response to activity.
+
+In the Events API, your Events API Request URL is the target location where all the events your connection is subscribed to will be delivered, regardless of the event type.
+
+Since your connected application will have only one Request URL, maybe you'll need to do an additional dispatch or routing server-side after receiving event data.
+
+Your Request URL will receive JSON-based payloads containing wrapped event types.
+
+## Performance considerations
```
Comment (fitn) [inline content/events-api/receiving-events.md:22]: I don't know if it's a good thing to write this part here. Maybe the dedicated "best practices / advices" section would be better?

My concern is that it's far more complex than what is written here and it's not only the volume of events but also the volume of API events we need to send. It's not only a "receiving events" topic. It's also about the batch size, the nightly sync, the queuing system, number of activated events subscription, and it brings some topics like limitations, retention duration, optimization
Also, there are a lot of customizable things that would help in solving problems

**Review by fitn — COMMENTED** [review]
A few comments and a part that I would move. We can have a discussion on it :)

File: content/events-api/overview.md [hunk 3]
```diff
@@ -2,6 +2,32 @@
 
 _All the essential things you need to know._
 
-## Scope of the Events API
+By using the Akeneo Events API, you will receive events on: product creation, product update, and product deletion. 
+The feature is working on simple products and product models and variants.
```
Comment (pierrejolly) [inline content/events-api/overview.md:6]:
```suggestion
The feature is working as well on simple products than on product models and variants.
```

**Review by pierrejolly — APPROVED** [review]

**Review by tomglvng — APPROVED** [review]

File: content/events-api/overview.md [hunk 4]
```diff
@@ -2,6 +2,32 @@
 
 _All the essential things you need to know._
 
-## Scope of the Events API
+By using the Akeneo Events API, you will receive events on: product creation, product update, and product deletion.
```
Comment (willy-ahva) [inline content/events-api/overview.md:5]:
```suggestion
By using the Akeneo Events API, you will receive events on the creation, update, and deletion of simple products, variant products, and product models. 
```

File: content/events-api/receiving-events.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+Be careful as some events can be sent in the wrong order, more than once, or get lost altogether. This is why the `event_id` and `event_datetime` are essential.
+
+- The `event_id` allows you to track the events you have already processed. You will not receive events older than **1 hours**, so no need to keep track of them passed that.
+- The `event_datetime` allows you to not processing an event if you receive an older event after a newer one.
+
+## Events API Request URLs
+
+Request URL receives an HTTP POST containing data in response to activity.
+
+In the Events API, your Events API Request URL is the target location. Akeneo PIM will deliver all the events, regardless of the event type, to this location
```
Comment (willy-ahva) [inline content/events-api/receiving-events.md:15]:
```suggestion
In the Events API, your Events API Request URL is the target location. Akeneo PIM will deliver all the events, regardless of the event type, to this location.
```

**Review by willy-ahva — APPROVED** [review]

File: content/events-api/overview.md [hunk 6]
```diff
@@ -2,6 +2,32 @@
 
 _All the essential things you need to know._
 
-## Scope of the Events API
+By using the Akeneo Events API, you will receive events on: product creation, product update, and product deletion. 
+The feature is working as well on simple products than on product models and variants.
```
Comment (meganelepalud-akeneo) [inline content/events-api/overview.md:6]:
```suggestion
The feature is working as well on simple products as on product models and variants.
```

**Pull request merged**